core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart

```diff
@@ -184,53 +228,7 @@ class _HtmlContentViewerOnWebState extends State<HtmlContentViewerOnWeb> {
       ..style.border = 'none'
       ..style.overflow = 'hidden'
       ..style.width = '100%'
-      ..style.height = '100%'
-      ..onLoad.listen((event) async {
```

Comment:
final subscription = iframe.onLoad.listen();
// ....


subscription.cancel();

Reply:
Leak still happens this way